collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+- Default base part of the Android package name was changed from `org.godotengine` to `com.example` ([GH-80761](https://github.com/godotengine/godot/pull/80761)).
+
+
+### Core and systems
+
+Following big changes to resource loading and multithreading done in Godot 4.1, the core team is more focused on stability this time around.
+
+This includes addressing design limitations when changing scenes or renaming nodes during `ready`, which led to crashes ([GH-78988](https://github.com/godotengine/godot/pull/78988), [GH-78706](https://github.com/godotengine/godot/pull/78706)), as well as various problems related to renaming and moving files around ([GH-80503](https://github.com/godotengine/godot/pull/80503), [GH-81657](https://github.com/godotengine/godot/pull/81657)). The latter could also lead to corruption of scene files, which should no longer occur. As should not occur sporadic changing of resources IDs in scenes ([GH-65011](https://github.com/godotengine/godot/pull/65011)). There is still room for improvement, but this already makes 4.2 way more version control friendly.
+
+These issues were fixed thanks to submissions from veteran contributors, such as [Pedro J. Estébanez](https://github.com/RandomShaper), [RedworkDE](https://github.com/RedworkDE), and [Rindbee](https://github.com/Rindbee), as well as project newcomers who only started contributing recently, [OXTyler](https://github.com/OXTyler) and [Jordyfel](https://github.com/Jordyfel).
+
+Sometimes a small mistake can result in major and unexpected differences in behavior. Such is the bug that has been well spotted and fixed by the project's bugsquad powerhouse [AThousandShips](https://github.com/AThousandShips) ([GH-81037](https://github.com/godotengine/godot/pull/81037)). This issue has been causing abnormal memory usage and performance issues in seemingly trivial use cases. If you are using a lot of plain Godot objects, you should clearly see the difference in 4.2.
+
+Smaller but notable feature additions have made their way into the release as well. You can now load OGG files at runtime, from a buffer or from a file path ([GH-78084](https://github.com/godotengine/godot/pull/78084) by [K. S. Ernest Lee](https://github.com/fire)). Same goes for SVG files, which can be loaded from a binary buffer or a string ([GH-78248](https://github.com/godotengine/godot/pull/78248) by [Felipe Augusto Marques](https://github.com/felaugmar)).
+
+![GodSVG is an SVG editor app made with Godot 4 by MewPurPur](/storage/blog/godot-4-2-beta/svg-editor-by-mewpurpur.webp)
+
+_[GodSVG](https://github.com/MewPurPur/GodSVG) is an SVG editor app made with Godot 4 by [MewPurPur](https://github.com/MewPurPur)._
+
+
+### 2D and 3D features
+
+Godot is a general purpose game engine, and features a variety of components aimed at helping you develop both 2D and 3D games. While not necessarily core, they are an integral part of the engine and thus get love an attention all the same in every release.
```

Comment:
```suggestion
Godot is a general purpose game engine, and features a variety of components aimed at helping you develop both 2D and 3D games. While not necessarily core, they are an integral part of the engine and thus get love and attention all the same in every release.
```
Are the 2D and 3D components not "core"?

Reply:
This split allows for more focused sections.